config/config_mgmt.py

```diff
@@ -71,18 +68,24 @@ def __init__(self, source="configDB", debug=False, allowExtraTables=True):
 
         return
 
-    def logInFile(self, header="", obj=None, json=False):
+    def logInFile(self, msg=None, logLevel='INFO'):
+
+        logLevelMap= {'CRIT'  : syslog.LOG_CRIT,
+                      'ERR'   : syslog.LOG_ERR,
+                      'INFO'  : syslog.LOG_INFO,
+                      'DEBUG' : syslog.LOG_DEBUG
+                   }
+        # return if loglevel is DEBUG. but debug is not set
+        if logLevel == 'DEBUG' and self.DEBUG == False:
+            return
+        # default logLevel is INFO
+        if logLevelMap.get(logLevel) == None:
+            logLevel='INFO'
 
-        if self.DEBUG_FILE:
-            with open(self.DEBUG_FILE, 'a') as df:
-                time = datetime.now()
-                df.write('\n\n{}: {}\n'.format(time, header))
-                if json:
-                    dump(obj, df, indent=4)
-                elif obj:
-                    #print(obj)
-                    df.write('{}: {}'.format(time, obj))
-                df.write('\n----')
+        if msg:
+            syslog.openlog(self.SYSLOG_IDENTIFIER)
```

Comment:
every message we have to open and close the log?

Reply:
I think, we shd open and close one time. I will change accordingly.